x-pack/plugins/ingest_manager/dev_docs/api_integration_tests.md

```diff
@@ -0,0 +1,107 @@
+# API integration tests
+
+Many API integration tests for Ingest Manager trigger at some point a connection to the package registry, and retrieval of some packages. If these connections are made to a package registry deployment outside of Kibana CI, these tests can fail at any time for two reasons:
+* the deployed registry is temporarily unavailable
+* the packages served by the registry do not match the expectation of the code under test
+
+For that reason, we run a dockerized version of the package registry in Kibana CI. For this to work, our tests must run against a custom test configuration and be kept in a custom directory, `x-pack/test/ingest_manager_api_integration`.
+
+## How to run the tests locally
+
+In the `x-pack` directory, run:
```

Comment:
I recently learned about some yarn scripts that I think are preferable using `node scripts/*`

For the two terminal approach listed shown here, in the kibana root, 

  * in one shell, run `yarn test:ftr:server --config x-pack/test/ingest_manager_api_integration/config.ts` 
  * in another, run `yarn test:ftr:runner --config x-pack/test/ingest_manager_api_integration/config.ts`

If you want to run everything in one go, this seems to work

`yarn test:ftr --config x-pack/test/ingest_manager_api_integration/config.ts`

Reply:
Thank you! I've updated the documentation accordingly.

Reply:
That's cool. I wasn't aware of that either. 👍